aws-ssm-patchbaseline/src/main/java/software/amazon/ssm/patchbaseline/HandlerWrapper.java

```diff
@@ -0,0 +1,192 @@
+// This is a generated file. Modifications will be overwritten.
+package software.amazon.ssm.patchbaseline;
+
+import com.amazonaws.AmazonServiceException;
+import software.amazon.awssdk.awscore.exception.AwsServiceException;
+import software.amazon.awssdk.regions.PartitionMetadata;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.cloudformation.Action;
+import software.amazon.cloudformation.exceptions.BaseHandlerException;
+import software.amazon.cloudformation.LambdaWrapper;
+import software.amazon.cloudformation.loggers.LambdaLogPublisher;
+import software.amazon.cloudformation.metrics.MetricsPublisher;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.CallbackAdapter;
+import software.amazon.cloudformation.proxy.HandlerErrorCode;
+import software.amazon.cloudformation.proxy.HandlerRequest;
+import software.amazon.cloudformation.proxy.LoggerProxy;
+import software.amazon.cloudformation.proxy.ProgressEvent;
+import software.amazon.cloudformation.proxy.RequestContext;
+import software.amazon.cloudformation.proxy.RequestData;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+import software.amazon.cloudformation.proxy.ResourceHandlerTestPayload;
+import software.amazon.cloudformation.resource.ResourceTypeSchema;
+import software.amazon.cloudformation.resource.SchemaValidator;
+import software.amazon.cloudformation.resource.Serializer;
+import software.amazon.cloudformation.scheduler.CloudWatchScheduler;
+import com.amazonaws.services.lambda.runtime.Context;
+import com.fasterxml.jackson.core.type.TypeReference;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.FileOutputStream;
+import java.io.FileReader;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.PrintStream;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import org.apache.commons.io.IOUtils;
+import org.json.JSONObject;
+
+
+public class HandlerWrapper extends LambdaWrapper<ResourceModel, CallbackContext> {
```

Comment:
why this and the following file?

Reply:
i see handlerWrapper is called in the template.json, but the whole packages worked fine without this file, or the App file, so whats the function of those files and template.json. 
also left a comment in template.json.

Reply:
The handlerWrapper class is a wrapper class which will be help us to invoke the handler events like (create, update, read, list, delete).
The template json file is used to test the sam & contract test senario's. for all the handler actions and the template.json file is the entry point to test the contract test changes so that i have declared the handler-wrapper class in the file.

If you have any conerns , Please let me know

Reply:
if it's for testing purpose, i think we can remove them.

Reply:
okay i am removing handlerWrapper files also.